doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/examples/ml-example.md

```diff
@@ -0,0 +1,139 @@
+(vm-ml-example)=
+
+# XGBoost-Ray on VMs
+
+:::{note}
+To learn the basics of Ray on VMs, we recommend taking a look
+at the {ref}`introductory guide<ref-cluster-quick-start-vms-under-construction>` first.
+:::
+
+
+In this guide, we show you how to run a sample Ray machine learning
+workload on AWS cloud.
+
+We will run Ray's {ref}`XGBoost training benchmark<xgboost-benchmark>` with a 100 gigabyte training set.
+To learn more about XGBoost-Ray, check out that library's {ref}`documentation<xgboost-ray>`.
```

Comment:
There was a PR that updated the XGBoost references. https://github.com/ray-project/ray/pull/27687/files
It's now
```
To learn more about using Ray's XGBoostTrainer, check out {ref}`the XGBoostTrainer documentation <train-gbdt-guide>`.
```